### ISSUE TITLE
Package not found error when trying to install from conda-forge

### ISSUE THREAD

**Issue by ethanwhite** [mzabrams/fars-cleaner#3]
The README says that the preferred installation method is using conda with the following command:

```
conda install -c conda-forge fars-cleaner
```

When attempting this using:

```
conda create -n fars-cleaner python=3
conda activate fars-cleaner
conda install -c conda-forge fars-cleaner
```

I get:

```
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - fars_cleaner

Current channels:

  - https://conda.anaconda.org/conda-forge/linux-64
  - https://conda.anaconda.org/conda-forge/noarch
  - https://repo.anaconda.com/pkgs/main/linux-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/linux-64
  - https://repo.anaconda.com/pkgs/r/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.
```

and I also can't find the package using the conda-forge search.

This is a review issue for https://github.com/openjournals/joss-reviews/issues/4678

**Comment by mzabrams** [comment]
This will resolve hopefully in the next few days -- waiting on a package version bump on `pyjanitor` in conda-forge to be able to package and upload properly.

**Comment by mzabrams** [comment]
Track issue at [conda-forge/pyjanitor-feedstock#57](https://github.com/conda-forge/pyjanitor-feedstock/pull/57)

**Comment by mzabrams** [comment]
Pending PR acceptance on condo-forge, this will be resolved -- [conda-forge/staged-recipes#20792](https://github.com/conda-forge/staged-recipes/pull/20792). Currently, package is available on conda and pypi

**Comment by mzabrams** [comment]
Closing with [conda-forge/staged-recipes#20792](https://github.com/conda-forge/staged-recipes/pull/20792) -- fars_cleaner now available on PyPi, conda, and conda-forge
